source/device_defender.c

```diff
@@ -466,12 +707,23 @@ static void s_reporting_task_fn(struct aws_task *task, void *userdata, enum aws_
     goto cleanup;
 
 cleanup:
+    s_clean_up_metric_data(custom_metric_data, defender_task);
 
     if (aws_hash_table_is_valid(&ifconfig.iface_name_to_info)) {
         aws_hash_table_clean_up(&ifconfig.iface_name_to_info);
     }
 }
 
+/**
+ * Creates DeviceDefender task based on the configuration structure details
+ * passed in, and immediately starts the task to recur until
+ * aws_iotdevice_defender_v1_stop_task() is called
+ *
+ * Any custom metrics must have been added to the configuration object before
+ * invoking this function.
+ *
+ *
```

Comment:
extra empty line

Reply:
I think I have this fixed. Recently converted to emacs so hopefully I'll get good at seeing/managing or autocorrecting this.